doc/rados/configuration/osd-config-ref.rst

```diff
@@ -4,11 +4,11 @@
 
 .. index:: OSD; configuration
 
-You can configure Ceph OSD Daemons in the Ceph configuration file (or in recent
-releases, the central config store), but Ceph OSD
-Daemons can use the default values and a very minimal configuration. A minimal
-Ceph OSD Daemon configuration sets ``osd journal size`` (for Filestore), ``host``,  and
-uses default values for nearly everything else.
+It is possible to configure Ceph OSD Daemons in the Ceph configuration file (or
+in recent releases, the central config store), but Ceph OSD Daemons can use the
```

Comment:
Can we remove "in the Ceph configuration file (or in recent releases"?

Reply:
Suggest now that we remove the parenthetical only, but also the last sentence on lines 9-11 below.  We no longer need to worry about Filestore.